src/Strategy/Composer.hs

```diff
@@ -72,6 +113,17 @@ instance ToJSON ComposerProject
 instance AnalyzeProject ComposerProject where
   analyzeProject _ = getDeps
 
+instance LicenseAnalyzeProject ComposerProject where
+  licenseAnalyzeProject = analyzeLicenses . composerJson
+
+analyzeLicenses :: (Has ReadFS sig m, Has Diagnostics sig m) => Maybe (Path Abs File) -> m [LicenseResult]
+analyzeLicenses Nothing = pure []
+analyzeLicenses (Just licenseFileName) = do
+  let mkLicenseResults l = [LicenseResult (toFilePath licenseFileName) l]
+      textToLicense = License LicenseSPDX
```

Comment:
Is this guaranteed that licences will always be in SPDX?

Ideally we can check against https://github.com/spdx/license-list-data, but this is likely out of scope for PR. Either way would prefer tiny comment.

Reply:
Probably a link to the composer docs and a short comment should suffice, the docs are pretty clear.

Reply:
The recommendation is that they be SPDX. I have added a link to the composer documentation to the code.

Reply:
👍